ini/stm32f1.ini

```diff
@@ -366,11 +365,31 @@ extra_scripts = ${common_stm32.extra_scripts}
 # FYSETC STM32F103RC
 #
 [env:STM32F103RC_fysetc]
-platform          = ${common_stm32.platform}
-extends           = common_STM32F103RC
-extra_scripts     = ${common_stm32.extra_scripts}
+platform             = ${common_stm32.platform}
+extends              = common_STM32F103RC
+extra_scripts        = ${common_stm32.extra_scripts}
   buildroot/share/PlatformIO/scripts/STM32F103RC_fysetc.py
-build_flags       = ${common_stm32.build_flags} -DDEBUG_LEVEL=0
-lib_ldf_mode      = chain
-debug_tool        = stlink
-upload_protocol   = serial
+build_flags          = ${common_stm32.build_flags} -DDEBUG_LEVEL=0
+lib_ldf_mode         = chain
+debug_tool           = stlink
+upload_protocol      = serial
+
+#
+# Longer 3D board in Alfawise U20 (STM32F103VET6)
+#
+[env:STM32F103VE_longer]
+platform             = ${common_stm32.platform}
+extends              = common_stm32
+board                = genericSTM32F103VE
+board_build.core     = stm32
+board_build.variant  = MARLIN_F103Zx
```

Comment:
MARLIN_F103Vx i guess here for the VE (100pins), not Z (144pins)

Reply:
Are we certain? A few boards ship with different chips in different batches.

Reply:
Ok, makes sense.

Reply:
Hmm, won't build because `pins_CHITU3D_V5.h` includes:
```cpp
#define FAN_PIN                             PG13  // FAN
#define FAN2_PIN                            PG14
```
…and these pins aren't defined for the smaller package.

Reply:
env:STM32F103VE_longer (not chitu) for some reason the comment moved in later commits...